config/demo_config.ini

```diff
@@ -1,17 +1,18 @@
 [PATHS]
 bam_file=
 reference_file=/oicr/data/genomes/homo_sapiens_mc/UCSC/hg19_random/Genomic/bwa/0.7.12/hg19_random.fa
-output_dir=/u/iwarikoo/Debarcer2/d_output/
+outdir=/u/iwarikoo/Debarcer2/d_output/
```

Comment:
i think this file is outdated, and can just be removed.  if not, remove hardcoded reference to home directory